docs/reference/ml/apis/dfanalyticsresources.asciidoc

```diff
@@ -7,34 +7,61 @@ A {dfanalytics} configuration object has the following properties:
 
 `analysis`::
   (object) The type of analysis that is performed on the `source`. For example: 
-  `outlier_detection`.
+  `outlier_detection`. For more information, see <<dfanalytics-types>>.
+  
+`analyzed_fields`::
+  (object) You can specify both `includes` and/or `excludes` patterns. If 
+  `analyzed_fields` is not set, only the relevant fileds will be included. For 
+  example all the numeric fields for {oldetection}.
 
 `dest`::
-  (object) The destination configuration of the analysis, consisting of `index` 
-  and optionally `results_field` (`ml` by default). 
+  (object) The destination configuration of the analysis. For more information, 
+  see <<dfanalytics-dest-resources>>.
 
 `id`::
   (string) The unique identifier for the {dfanalytics-job}. This identifier can 
   contain lowercase alphanumeric characters (a-z and 0-9), hyphens, and 
   underscores. It must start and end with alphanumeric characters. This property 
   is informational; you cannot change the identifier for existing jobs.
+  
+`model_memory_limit`::
+  (long or string) The approximate maximum amount of memory resources that are 
+  required for analytical processing. Once this limit is approached, data 
+  pruning becomes more aggressive. Upon exceeding this limit, new entities are 
+  not modelled. The default value for {dfanalytics-jobs} is `1gb`.
+  +
+--
+If you specify a number instead of a string, the units are assumed to be MiB.
+Specifying a string is recommended for clarity. If you specify a byte size unit
+of `b` or `kb` and the number does not equate to a discrete number of megabytes,
+it is rounded down to the closest MiB. The minimum valid value is 1 MiB. If you
```

Comment:
Is this imposed on the Java side @dimitris-athanasiou? The rounding and minimum limit do not happen on the C++ side.

Reply:
No, there is no rounding happening in java side either. We're simply converting the value to `bytes` and pass it to the C++ process.